### DISCUSSION TITLE
InnateMC website

### DISCUSSION THREAD

**Discussion by BoogieMonster1O1** [InnateMC/InnateMC#29] [RFCs]
I propose the creation of a website for InnateMC. The website will serve as a promotional platform for the launcher and will provide additional information to potential users. The following pages will be included on the website:
- Home Page: An overview of InnateMC, including its features, system requirements, a download link, and ways to contribute.
- Documentation Page: Instructions for installing InnateMC as well as troubleshooting advice.
- Support Page: This page will include links to the discord server for tech support, and GitHub for reporting issues and requesting features.
- Blog Page: This page will contain related blog posts, such as updates, new features, and community news.

The development of a website for InnateMC will result in the following advantages:
- The website will serve as a platform for InnateMC to be promoted to potential users. 
- The website will include more information about InnateMC, such as features, the development process, and ways to contribute. This may hopefully attract potential contributors to help out with the project.
- The documentation page will assist users in installing and using Launcher, and allow users to easily report issues to GitHub.
- The blog page will aid in building a community around Launcher and also keep them up to date on new features and updates.

The website will be built with a modern web framework like Next.js or SvelteKit hosted as a stateless function, or a static site generator hosted on GitHub pages. If we ever decide to use Objective-C in InnateMC for anything, we must use ember.js as the web framework.

The development of a website for InnateMC will hopefully boost the release. Hopefully, it can attract potential contributors. Users can also easily discover the features and requirements of InnateMC, as well as get installation and usage support. Overall, a website for InnateMC can significantly increase the launcher's visibility.

**Comment by heftymouse** [comment]
I'd suggest using Astro instead of Next or SvelteKit - it's primarily a static site generator, but it can dynamically render certain pages if configured to do so. It has a great developer experience and creates lightweight sites.

Also, acquiring a (non-Freenom) domain should be a top priority; as for what it should be I have no idea
